src/libpmem2/map.c

```diff
@@ -131,55 +131,56 @@ pmem2_validate_offset(const struct pmem2_config *cfg, size_t *offset,
 	return 0;
 }
 
-static struct ravl_interval *ri;
-static os_rwlock_t lock;
-
 /*
  * mapping_min - return min boundary for mapping
  */
 static size_t
-mapping_min(void *map)
+mapping_min(void *addr)
 {
-	return (size_t)pmem2_map_get_address(map);
+	struct pmem2_map *map = (struct pmem2_map *)addr;
+	return (size_t)map->addr;
 }
 
 /*
  * mapping_max - return max boundary for mapping
  */
 static size_t
-mapping_max(void *map)
+mapping_max(void *addr)
 {
-	return (size_t)pmem2_map_get_address(map) +
-		pmem2_map_get_size(map);
+	struct pmem2_map *map = (struct pmem2_map *)addr;
+	return (size_t)map->addr + map->content_length;
 }
 
+static struct pmem2_state {
+	struct ravl_interval *range_map;
+	os_rwlock_t range_map_lock;
+} State;
+
 /*
- * pmem2_map_init -- initialize the map module
+ * pmem2_state_init -- initialize the state module
```

Comment:
Revert this name change, I thought there would be more in the `pmem2_map_init`.

Reply:
Done and rebased.